Library/Homebrew/cli/named_args.rb

```diff
@@ -78,17 +78,28 @@ def to_formulae_and_casks(
         @to_formulae_and_casks ||= T.let(
           {}, T.nilable(T::Hash[T.nilable(Symbol), T::Array[T.any(Formula, Keg, Cask::Cask)]])
         )
-        @to_formulae_and_casks[only] ||= downcased_unique_named.flat_map do |name|
-          options = { warn: }.compact
-          load_formula_or_cask(name, only:, method:, **options)
-        rescue FormulaUnreadableError, FormulaClassUnavailableError,
-               TapFormulaUnreadableError, TapFormulaClassUnavailableError,
-               Cask::CaskUnreadableError
-          # Need to rescue before `*UnavailableError` (superclass of this)
-          # The formula/cask was found, but there's a problem with its implementation
-          raise
-        rescue NoSuchKegError, FormulaUnavailableError, Cask::CaskUnavailableError, FormulaOrCaskUnavailableError
-          ignore_unavailable ? [] : raise
+        @to_formulae_and_casks[only] ||= begin
+          download_queue = Homebrew::DownloadQueue.new if Homebrew::EnvConfig.download_concurrency > 1
```

Comment:
We do this enough I almost wonder if we want a `Homebrew::DownloadQueue.new_if_needed` or something that does the `download_concurrency` check internally 🤔

Reply:
Will do this in a follow-up PR